content/docs/esc/integrations/dynamic-secrets/aws-parameter-store.md

```diff
@@ -0,0 +1,104 @@
+---
+title: aws-parameter-store
+title_tag: aws-parameter-store Pulumi ESC Provider
+meta_desc: The aws-parameter-store Pulumi ESC Provider enables you to dynamically import parameters from AWS Systems Manager - Parameter Store.
+h1: aws-parameter-store
+menu:
+  esc:
+    identifier: aws-parameter-store
+    parent: esc-dynamic-secrets
+    weight: 1
+aliases:
+    - /docs/pulumi-cloud/esc/providers/aws-parameter-store/
+    - /docs/esc/providers/aws-parameter-store/
+---
+
+The `aws-parameter-store` provider enables you to dynamically import parameters from AWS Systems Manager - Parameter Store into your Environment. The provider will return a map of names to parameters.
+
+## Example
+
+```yaml
+aws:
+  login:
+    fn::open::aws-login:
+      oidc:
+        roleArn: arn:aws:iam::123456789:role/esc-oidc
+        sessionName: pulumi-environments-session
+  params:
+    fn::open::aws-parameter-store:
+      region: us-west-1
+      login: ${aws.login}
+      get:
+        myKey:
+          name: /myNamespace/myKey
+        myKeyByVersion:
+          name: /myNamespace/myKey:1
+        myKeyByVersionLabel:
+          name: /myNamespace/myKey:stable
+        secureKey:
+          name: /myNamespace/secureKey
+          decrypt: true
+        myList:
+          name: /myNamespace/myList
+
+```
+
+## Configuring OIDC
+
+To learn how to configure OpenID Connect (OIDC) between Pulumi Cloud and AWS, see the [OpenID Connect integration](/docs/pulumi-cloud/oidc/provider/aws/) documentation. Once you have completed these steps, you can validate that your configuration is working by running either of the following:
+
+* `esc open <your-org>/<your-environment>` command of the [Pulumi ESC CLI](/docs/esc-cli/)
+* `pulumi env open <your-org>/<your-environment>` command of the [Pulumi CLI](/docs/install/)
```

Comment:
```suggestion
* `esc open <org>/<project>/<environment>` command of the [Pulumi ESC CLI](/docs/esc-cli/)
* `pulumi env open <org>/<project>/<environment>` command of the [Pulumi CLI](/docs/install/)
```

Reply:
nice callout. i'll actually update this in the other pages too